rotkehlchen/utils/data_structures.py

```diff
@@ -33,3 +33,32 @@ def remove(self, key: str) -> None:
     def clear(self) -> None:
         """Delete all entries in the cache"""
         self.cache.clear()
+
+
+class LRUSetCache(Generic[RT]):
+    """
+    LRU cache that works like a set.
+    Internally it uses an OrderedDict to keep the order that maps to None.
```

Comment:
I don't get it. Why did you use ordered dict internally? I don't understand what `"to keep the order that maps to None."` means

You probably mean: In order to be able to keep the order of insertion so that the LRU mechanism can function. Right?

Can you rephrase?